ansible/main.yml

```
@@ -0,0 +1,110 @@
+---
+- hosts: all
+  vars:
+    app_path: /home/ubuntu/RD-WATCH
+    venv: "{{ app_path }}/venv"
```

```suggestion
    venv: "{{ ansible_facts['user_dir'] }}/venv"
```

Let's not put the virtualenv inside the source repo.